en/04_Changelogs/5.0.0.md

```diff
@@ -26,6 +28,53 @@ title: 5.0.0 (unreleased)
+[cwp/cwp-search](https://packagist.org/packages/cwp/cwp-search) | CMS5 comptabile without support | 
+[hafriedlander/phockito](https://github.com/hafriedlander/phockito) | Dropped | 
+[hafriedlander/silverstripe-phockito](https://packagist.org/packages/hafriedlander/silverstripe-phockito) | Dropped | 
+[lekoala/silverstripe-debugbar](https://packagist.org/packages/lekoala/silverstripe-debugbar) | CMS5 comptabile without support | Debug bar is a development tool that should not be installed in production.
+[silverstripe/activedirectory](https://packagist.org/packages/silverstripe/activedirectory) | Dropped | 
+[silverstripe/akismet](https://packagist.org/packages/silverstripe/akismet) | Dropped | The parent library is outdated and they are better alternatives like [UndefinedOffset/silverstripe-nocaptcha](https://packagist.org/packages/undefinedoffset/silverstripe-nocaptcha)
+[silverstripe/behat-extension](https://github.com/silverstripe/silverstripe-behat-extension) | CMS5 comptabile without official support | 
+[silverstripe/ckan-registry](https://packagist.org/packages/silverstripe/ckan-registry) | CMS5 comptabile without support | 
+[silverstripe/comment-notifications](https://packagist.org/packages/silverstripe/comment-notifications) | CMS5 comptabile without official support | 
+[silverstripe/comments](https://packagist.org/packages/silverstripe/comments) | CMS5 comptabile without official support | 
+[silverstripe/content-widget](https://packagist.org/packages/silverstripe/content-widget) | CMS5 comptabile without support | 
+[silverstripe/fulltextsearch](https://packagist.org/packages/silverstripe/fulltextsearch) | CMS5 comptabile without support | 
+[silverstripe/graphql-devtools](https://packagist.org/packages/silverstripe/graphql-devtools) | CMS5 comptabile without official support | 
+[silverstripe/html5](https://packagist.org/packages/silverstripe/html5) | Folded back in core | 
+[silverstripe/postgresql](https://packagist.org/packages/silverstripe/postgresql) | CMS5 comptabile without support | 
+[silverstripe/recipe-ccl](https://github.com/silverstripe/recipe-ccl) | CMS5 comptabile without support | 
+[silverstripe/recipe-solr-search](https://github.com/silverstripe/recipe-solr-search) | CMS5 comptabile without support | 
+[silverstripe/spellcheck](https://packagist.org/packages/silverstripe/spellcheck) | Dropped | 
+[silverstripe/sqlite3](https://packagist.org/packages/silverstripe/sqlite3) | CMS5 comptabile without support | 
+[silverstripe/widgets](https://packagist.org/packages/silverstripe/widgets) | CMS5 comptabile without official support | 
+[tijsverkoyen/akismet](https://github.com/tijsverkoyen/akismet) | Dropped | 
+[tractorcow/classproxy](https://github.com/tractorcow/classproxy) | CMS5 comptabile without official support | 
+[tractorcow/silverstripe-proxy-db](https://packagist.org/packages/tractorcow/silverstripe-proxy-db) | CMS5 comptabile without official support | 
+
+## Fixed dependencies {#fixed-depdencies}
```

Comment:
Remove until needed so we don't forget it here empty?

```suggestion
```

Reply:
I'll add a link to the full list below.